Makefile

```diff
@@ -139,7 +158,7 @@ fluentd:
 image-cleanup-test-images:
 	cd misc/image-cleanup-test-images; $(MAKE) $(MFLAGS)
 
-get-deps:
+get-deps: get-cni-sources
```

Comment:
Do we really need this here? I saw `get-deps` is only executed when running the test, where `amazon-ecs-cni-plugins` isn't required.